matter_server/config.yaml

```diff
@@ -11,7 +11,7 @@ arch:
 discovery:
   - matter
 hassio_api: true
-homeassistant: 2023.2.0b0
+homeassistant: 2023.2.5
```

Comment:
Why did we change the required Home Assistant Core version? This change is a breaking change now.

Reply:
Because only in that version the client changes got in? This was advised by @marcelveldt